packages/rive/src/RemotionRiveCanvas.tsx

```diff
@@ -96,12 +98,13 @@ export const RemotionRiveCanvas: React.FC<RiveProps> = ({
 						artboard,
 						renderer,
 					});
+					onLoad(riveCanvasInstance);
 				});
 			})
 			.catch((newErr) => {
 				setError(newErr);
 			});
-	}, [animationIndex, artboardName, riveCanvasInstance, src]);
+	}, [animationIndex, artboardName, riveCanvasInstance, src, onLoad]);
```

Comment:
Thanks a lot for the PR!

This might be problematic, this useEffect will trigger on every React re-render if the function is not memoized. The example should wrap it in useCallback() to fix it and we should not use `() => {}` as a default, which will also lead to more more re-renders.

Don't worry about the tests, I will get them running.

Reply:
Thank YOU for putting together this amazing project 😍 . What's your suggestion for the default value for `onLoad`, should I allow it to be null in which case have an `if` statement to avoid calling the callback?

Reply:
😊

Yes, it can be null and then be wrapped in an `if`, that is fine!

Reply:
PR updated, let me know if this what you had in mind. Happy to do as many changes as required 💪  I think this is my first contribution to a React-based project and I'm not even a frontend developer so I barely know what I'm doing 😅

Reply:
Appreciate it! Will now process it until everything is green 😊